### DISCUSSION TITLE
Converter

### DISCUSSION THREAD

**Discussion by marko-lazic** [ozlerhakan/poiji#205] [Q&A]
Hello,

Is there a way to convert cell values into objects other than basic types? Let's say the cell value should actually be converted to enum. Or cell value represents an id and the repository should find that entity and load it from the database. What's the best approach here? Should I write one model only for xls sheet and another one representing my business logic?

**Comment by ozlerhakan** [comment]
Hi @marko-lazic ,

Poiji currently only support basic types conversion. Feel free to contribute the project to make this possible :)
